lib/innit-server/Cargo.toml

```diff
@@ -13,17 +13,23 @@ innit-core = { path = "../../lib/innit-core" }
 si-data-ssm = { path = "../../lib/si-data-ssm" }
 si-settings = { path = "../../lib/si-settings" }
 si-std = { path = "../../lib/si-std" }
+si-tls = { path = "../../lib/si-tls" }
 telemetry = { path = "../../lib/telemetry-rs" }
 
 axum = { workspace = true }
+axum-server = { workspace = true }
```

Comment:
I believe this crate isn't being used anywhere as I don't see it in init-server's `BUCK` file, is that right?